packages/mobile/src/screens/new-item/components/new-auction-form/new-auction-form.tsx

```diff
@@ -0,0 +1,313 @@
+import React, { FC } from 'react';
+import { ColorPalette, IPostForms } from '@vse-bude/shared';
+import {
+  DropDown,
+  Input,
+  InfoIcon,
+  Text,
+  TouchableOpacity,
+  View,
+  Popover,
+  DateTimePicker,
+  ClockIcon,
+  CalendarIcon,
+} from '~/components/components';
+import { useAppForm, useTranslation, useState } from '~/hooks/hooks';
+import { CALLING_CODE, CITIES, COUNTRIES } from '~/mock/new-item';
+import { globalStyles } from '~/styles/styles';
+import dayjs from 'dayjs';
+
+import { AddPhotos } from '../add-photos/add-photos';
+
+import { useStyles } from './styles';
+
+const NewAuctionForm: FC = () => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+
+  const today = new Date();
+  const tomorrow = new Date();
+  tomorrow.setDate(today.getDate() + 1);
+  const [date, setDate] = useState(tomorrow);
+  const [time, setTime] = useState(tomorrow);
+  const [openDate, setOpenDate] = useState(false);
+  const [openTime, setOpenTime] = useState(false);
+
+  const { control, errors, setValue } = useAppForm<IPostForms>({
+    defaultValues: {
+      country: 'Ukraine',
+      callingCode: 'UA',
+    },
+  });
+
+  return (
+    <View>
+      <Text style={[globalStyles.fs14, globalStyles.mt5, styles.title]}>
+        {t('make_a_post.DOWNLOAD_PHOTOS')}
+      </Text>
+      <AddPhotos />
+      <Text style={[globalStyles.fs14, globalStyles.mt5, styles.title]}>
+        {t('make_a_post.DESCRIPTION')}
+      </Text>
+      <Input
+        label={t('make_a_post.CATEGORY')}
+        placeholder={t('make_a_post.CATEGORY_PLACEHOLDER')}
+        name="category"
+        control={control}
+        errors={errors}
+        contentContainerStyle={globalStyles.mt5}
+      />
+
+      <Input
+        label={t('make_a_post.TITLE_NAME')}
+        placeholder={t('make_a_post.TITLE_NAME_PLACEHOLDER')}
+        name="title"
+        control={control}
+        errors={errors}
+        contentContainerStyle={globalStyles.mt5}
+      />
+
+      <TouchableOpacity
+        onPress={() => {
+          setOpenDate(true);
+        }}
+      >
+        <Input
+          label={t('make_a_post.ENDING_DATE')}
+          placeholder={'-/-/-'}
+          name="endDate"
+          control={control}
+          errors={errors}
+          editable={false}
+          contentContainerStyle={globalStyles.mt5}
+        />
+        <CalendarIcon
+          size={22}
+          color={ColorPalette.GRAY_300}
+          style={[styles.inputIcon]}
+        />
+      </TouchableOpacity>
+      {openDate && (
+        <DateTimePicker
+          value={date}
+          mode={'date'}
+          is24Hour={true}
+          display="default"
+          minimumDate={tomorrow}
+          onChange={(_, selectedDate) => {
+            setOpenDate(false);
+            const currentDate = dayjs(selectedDate).format('DD/MM/YYYY');
+            setDate(selectedDate ?? tomorrow);
+            setValue('endDate', currentDate);
+          }}
+        />
```

Comment:
This should be extracted to a separate controlled component. In that case you won't need to use `setValue` explicitly